webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9160,6 +9160,9 @@ var CURRENT_GROUP_DATA={};
                     var resp = $applozic.parseJSON(obj.body);
                     typeof resp.message == "object" && $mck_msg_inner.data('last-message-received-time', resp.message.createdAtTime);
                     var messageType = resp.type;
+                    if (typeof resp.message == "object" && resp.message.message === 'Status changed to Resolved'){
```

Comment:
@akshatdhabi is there any other way to check than using "Status changed to Resolved'?
what if we later add localization and then text is changed to some other language.

Reply:
@devashishmamgain  changed the check to use message metadata instead of the message